modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the form view
+
+You can use a similar procedure to customize the following UI items:
+** Quick starts visible to users
+** Cluster roles accessible to a project
+** Actions visible on the +Add page
+
+.Prerequisites
+
+* You must have administrator privileges.
```

Comment:
What kind of admin privileges are these? Are they bound to a specific platform or a product?

Reply:
As far as I think they are just basic administrator user privileges. @debsmita1 - could you help me with this query please?

Reply:
The wording is very vague. This is how our documentation handles similar prerequisites: https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.2/html/administration_guide/installing-devspaces#installing-devspaces-on-openshift-using-the-cli-management-tool

Reply:
I am only reusing stuff from our content @deerskindoll. Kindly see the following link:

https://docs.openshift.com/container-platform/4.11/web_console/web-console.html#enable-multi-cluster-console_web-console

If you still feel it is vague, I will change it in my content to the format you suggested in the link as follows:

An Open Shift web console session by a cluster administrator.

Reply:
Fixed.